pydrive2/test/test_file.py

```diff
@@ -493,9 +521,10 @@ def test_Gfile_Conversion_Add_Remove_BOM(self):
     finally:
       self.cleanup_gfile_conversion_test(file1, file_name, downloaded_file_name)
 
+
   def test_InsertPrefix(self):
     # Create BytesIO.
-    file_obj = BytesIO('abc')
+    file_obj = BytesIO('abc'.encode('utf8'))
```

Comment:
is it py2 vs py3 difference?

Reply:
Not sure from where it comes exactly, but `BytesIO` doesn't accept plain string anymore and complains with error `TypeError: a bytes-like object is required, not 'str'`

Reply:
kk 👍